c/ntdrivers/diskperf.i.cil-1.c

```diff
@@ -3432,7 +3432,6 @@ int main(void)
   devobj.DeviceLock = kevent_1;
   devobj.SectorSize = __VERIFIER_nondet_int();
   devobj.Spare1 = __VERIFIER_nondet_int();
-  devobj.DeviceObjectExtension = (struct _DEVOBJ_EXTENSION *)0;
   devobj.Reserved = 0;
```

Comment:
In line 3368 `devobj.DeviceExtension` is allocated via `malloc`. As far as I see these are two different fields.

Reply:
Thank you, I must have been rather tired when working on this. This is now fixed.

Reply:
Still, I do not see where is this field used in the code (and thus, what this change fixes)?

Reply:
Well, I guess I keep being tired. It took quite a few more fixes (and none of them was the above). Now at least CBMC tells me that there aren't memory-safety issues left (unwinding loops up to 40 times). My apologies for the chaos.